docs/docs/gatsby-cli.md

```diff
@@ -9,27 +9,24 @@ Gatsby command line tool. This doc exists in [READme form in GitHub here](https:
 Lets you create new Gatsby sites using
 [Gatsby starters](https://www.gatsbyjs.org/docs/gatsby-starters/).
 
-It also lets you run commands on sites. The tool runs code from the `gatsby`
-package installed locally.
-
-## Install
+## How to use
 
-`npm install --global gatsby-cli`
+By using [npx](https://www.npmjs.com/package/npx) you can utilise the Gatsby command line tool without installing it globally.
 
-## How to use
+For example you can run `npx gatsby new your-site-name` to generate a new Gatsby site.
 
-Run `gatsby --help` for full help.
+Run `npx gatsby --help` for full help.
 
 ### New
 
-`gatsby new gatsby-site`
+`npx gatsby new gatsby-site`
 
 See the [Gatsby starters docs](https://www.gatsbyjs.org/docs/gatsby-starters/)
 for more.
 
 ### Develop
 
-At the root of a Gatsby site run `gatsby develop` to start the Gatsby
+At the root of a Gatsby site run `npm run develop` to start the Gatsby
```

Comment:
What about something like this?

```suggestion
At the root of a Gatsby site run `gatsby develop` (typically exposed with a package.json script runnable with `npm run develop`) to start the Gatsby
```

Reply:
We can't _guarantee_ that the starter/app template that the user is using _always_ contains these scripts, so it's nice to document it in a way that is a little more resilient to that.